src/components/Comments.jsx

```diff
@@ -0,0 +1,90 @@
+import { useEffect, useState } from 'react'
+import { getComments } from '../api'
+function Comments({ article_id }) {
+	const [comments, setComments] = useState([])
+	useEffect(() => {
+		getComments(article_id).then((data) => {
+			setComments(data)
+		})
+	}, [])
+	const months = [
+		'Jan',
+		'Feb',
+		'Mar',
+		'Apr',
+		'May',
+		'Jun',
+		'Jul',
+		'Aug',
+		'Sep',
+		'Oct',
+		'Nov',
+		'Dec',
+	]
+	function getFullDate(date) {
+		const commentedAt = new Date(date)
+		return `${commentedAt.getDate()} ${
+			months[commentedAt.getMonth()]
+		} ${commentedAt.getFullYear()}`
+	}
```

Comment:
There's a lot of logic here that you could make shorter at some point. Have a look at a library called moment.js to help with formatting the date. It isn't urgent so please don't feel you have to look at it right now